ca/certificate-authority.go

```diff
@@ -596,7 +596,17 @@ func (ca *CertificateAuthorityImpl) IssueCertificate(csr x509.CertificateRequest
 	}
 
 	// Submit the certificate to any configured CT logs
-	go ca.Publisher.SubmitToCT(certDER)
+	go func() {
```

Comment:
`publisher` will internally log any error it hits so this will cause duplicate log lines. Either this or the internal logging on the other side should be removed.